pages/windows/eventcreate.md

```diff
@@ -0,0 +1,19 @@
+# eventcreate
+
+> Create custom entries in the event log.
+
+- Create a new event in the log:
+
+`eventcreate /t {{success|error|warning|information}} /id {{id}} /d "{{message}}"`
```

Comment:
Where does the id come from ?

Reply:
It can be any number between 1 and 1000. You just choose it afaik.

Reply:
Ah, ok. So how about `Create a new event with a given id(1-1000) in the log` ?